templates/patient/insurance/_insurance_edit_screen_edit.html.twig

```diff
@@ -0,0 +1,161 @@
+{#
+Partial twig file that represents the insurance edit screen for a patient
+
+@package   OpenEMR
+@link      http://www.open-emr.org
+@author    Stephen Nielson <snielson@discoverandchange.com>
+@copyright Copyright (c) 2024 Care Management Solutions, Inc. <stephen.waite@cmsvt.com>
+@license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+#}
+<div id="edit-policy-screen" class="insuranceEditContainer d-none">
+    <ul class="tabNav">
+        {% for instype in insuranceTypes %}
+            <li class="{% if instype == activeType %}current{% endif %} nav-link-insurance-type-container"><a href="#" class="nav-link-insurance-type" data-type="{{ instype|attr }}">{{ instype|text|capitalize }}</a></li>
```

Comment:
just to be sure, would do the text after the capitalize

Reply:
Fixed